test/integration/statistics_announcement_test.rb

```diff
@@ -0,0 +1,35 @@
+require 'test_helper'
+
+class StatisticsAnnouncementTest < ActionDispatch::IntegrationTest
+
+  test "official statistics are rendered correctly" do
+    setup_and_visit_content_item('official_statistics')
+
+    assert_has_component_title(@content_item["title"])
+    assert page.has_text?(@content_item["description"])
+    assert_has_component_metadata_pair("Release date", "20 January 2016 9:30am (confirmed)")
+  end
+
+  test "national statistics are rendered correctly" do
+    setup_and_visit_content_item('national_statistics')
+
+    assert_has_component_title(@content_item["title"])
+    assert page.has_text?(@content_item["description"])
+    assert page.has_css?('.national-statistics-logo img')
+    assert_has_component_metadata_pair("Release date", "January 2016 (provisional)")
+  end
+
+  test "cancelled statistics are rendered correctly" do
+    setup_and_visit_content_item('cancelled_official_statistics')
+
+    assert_has_component_title(@content_item["title"])
+    assert page.has_text?(@content_item["description"])
+    within ('.cancellation-notice') do
+      assert page.has_text?('Statistics release cancelled'), "is cancelled"
+      assert page.has_text?(@content_item["details"]["cancellation_reason"]), "displays cancelleation reason"
```

Comment:
It might be safer to use `@content_item["details"].fetch("cancellation_reason")` so that we at least know if the `cancellation_reason` key is removed.